tests/composite-proofs/msg-js-obj/bbs.spec.ts

```diff
@@ -0,0 +1,589 @@
+import { initializeWasm } from '@docknetwork/crypto-wasm';
```

Comment:
This file can be removed if `scheme.ts` is used in `bbs-plus.spec.ts` in this directory.